src/material-examples/sidenav-overview/sidenav-overview-example.html

```diff
@@ -1,5 +1,5 @@
 <mat-sidenav-container class="example-container" *ngIf="shouldRun">
-  <mat-sidenav mode="side" opened="true">Sidenav content</mat-sidenav>
+  <mat-sidenav mode="side" [opened]="true">Sidenav content</mat-sidenav>
```

Comment:
This can actually just be `opened` with no assignment
```html
<mat-sidenav mode="side" opened>
```

Reply:
Yeah, you're right! 
But maybe it should be more verbose for the sake of clarity? Brief search showed that it is common for the docs to have `[bSome]="true"` rather than just `bSome`.

Reply:
In general I prefer to use the short form, otherwise people will interpret the examples as _needing_ to be written out the long way.

Reply:
Can I interpret that as a suggestion to fix every example in the docs according to your preferences?